cogs/welcome.py

```diff
@@ -26,29 +26,17 @@ async def on_voice_state_update(self, member, before, after):
         """Plays a welcome sound message when someone joins the channel."""
         if before.channel == None and after.channel != None and not member.bot and not after.self_deaf:
             now = arrow.now('America/Sao_Paulo')
-            userList = []
-            for onlineMember in after.channel.members:
-                user = session.query(User).filter_by(member_id=onlineMember.id).first()
-                if not user:
-                    user = User(
-                        member_id=onlineMember.id,
-                        guild_id=after.channel.guild.id,
-                        name=onlineMember.name,
-                        last_seen=now
-                        )
-                    session.add(user)
-                    userList.append(user)
-                    continue
-                
-                if (now - user.last_seen).total_seconds() < 3600 * 12:
-                    continue
-                elif not onlineMember.voice.self_deaf:
-                    userList.append(user)
-                
-
-            if userList:
+            user = session.query(User).filter_by(member_id=member.id).first()
```

Comment:
Pq vc removeu toda logica do userList?

Reply:
Pô, pq eu achei tosco e inútil. Tipo, era muito situacional. Do jeito que estava, só iria fazer alguma diferença em relação a não ter nada caso:
- existisse uma pessoa A, logada em uma sala do discord, não mutada, tendo ultrapassado o threshold de 12 horas sem receber bom dia;
- e entrasse uma pessoa B, pronta para receber bom dia, no mesmo canal da pessoa A.

Fora esse caso específico, ambos os códigos funcionam do mesmo jeito.

Reply:
achou errado